frontend/src/views/AccountView.vue

```diff
@@ -46,6 +48,13 @@ export default {
     RosalutionHeader,
     SectionBox,
   },
+  data() {
```

Comment:
`authStore.getUser()` is being called twice in this view, when it ought to be just called once.   This will need to get removed.  I understand it is necessary, so that the 'secretValue' in data can be set.  However, 'secretValue' needs to be migrated to be a VueJS 'computed' value in order to work best with VueJS' reactivity and rendering.

For data, there can be a value such as "showSecretValue" which is used as a boolean toggle in the 'secretValue' computed value to determine what text will get rendered.

This will also clean up a lot of the code for the methods below for toggleSecret and 'updateSecretValue'.

Reply:
Looks good. Much cleaner and less to maintain.